external-crates/move/crates/move-compiler/src/command_line/compiler.rs

```diff
@@ -278,8 +290,19 @@ impl<'a> Compiler<'a> {
             known_warning_filters,
             package_configs,
             default_config,
+            vfs,
         } = self;
-        generate_interface_files_for_deps(
+        let vfs = match vfs {
+            Some(vfs) => vfs,
+            None => VfsPath::new(PhysicalFS::new("/")),
```

Comment:
Is this... a physical filesystem from root?

Reply:
Yes. Some paths are provided as absolute paths right off the bat so I think this is the only way...

Reply:
SHouldn't we just map each path into the VfsPath? i.e. taking all of the input paths here and mapping `VfsPath::new(PhysicalFS::new`